scio-google-cloud-platform/src/main/scala/com/spotify/scio/bigquery/client/BigQueryConfig.scala

```diff
@@ -70,6 +72,14 @@ object BigQueryConfig {
   def readTimeoutMs: Option[Int] =
     BigQuerySysProps.ReadTimeoutMs.valueOption.map(_.toInt)
 
+  def impersonationInfo: Option[ImpersonationInfo] = {
+    BigQuerySysProps.ActAs.valueOption.map { actAs =>
+      val lifetime =
+        BigQuerySysProps.ImpersonationLifetimeSec.valueOption.flatMap(x => Try(x.toInt).toOption)
+      ImpersonationInfo(actAs, lifetime.getOrElse(3600))
```

Comment:
```suggestion
      ImpersonationInfo(actAs, lifetime.getOrElse(0))
```

I would set the default here to 0, that seems to be the Google library's hint to use its default value: https://github.com/googleapis/google-auth-library-java/blob/main/oauth2_http/java/com/google/auth/oauth2/ImpersonatedCredentials.java#L679